### ISSUE TITLE
Error /f perm set <perm> <faction>

### ISSUE THREAD

**Issue by ghost** [magnusulf/Factions#11]
## Affected Plugin

- **Affected Plugin Name:** Factions
- **Affected Plugin Version:** 3.1.2

## Describe your Environment

- **Server Version:** Spigot 1.13.2

## Steps to Reproduce this Bug

1. If I want to add permissions to another faction, it will return error in console and permissions do not work.
```
[22:48:28] [Server thread/INFO]: Uran234 issued server command: /f perm set container killkill yes
[22:48:28] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f perm set container killkill yes' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1636) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1476) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_191]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.NullPointerException: oid
	at com.massivecraft.massivecore.store.SenderColl.get(SenderColl.java:57) ~[?:?]
	at com.massivecraft.massivecore.command.type.sender.TypeSenderEntity.getResultForSenderId(TypeSenderEntity.java:62) ~[?:?]
	at com.massivecraft.massivecore.command.type.sender.TypeSenderEntity.getResultForSenderId(TypeSenderEntity.java:8) ~[?:?]
	at com.massivecraft.massivecore.command.type.sender.TypeSenderIdAbstract.read(TypeSenderIdAbstract.java:127) ~[?:?]
	at com.massivecraft.factions.cmd.type.TypeMPermable.read(TypeMPermable.java:79) ~[?:?]
	at com.massivecraft.factions.cmd.CmdFactionsPermSet.perform(CmdFactionsPermSet.java:40) ~[?:?]
	at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1177) ~[?:?]
	at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1133) ~[?:?]
	at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1133) ~[?:?]
	at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:60) ~[?:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:139) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
	... 14 more
```

2. If the player name and the faction name are the same, the player is set. Maybe something like 
```
/f perm set <perm> f <faction>...
/f perm set <perm> p <player>...
```

**Comment by magnusulf** [comment]
First of all, sincerely thank you, because of users like you who report bugs and do so properly I am able to fix them 😃 
Second, if you have multiple issues please make multiple tickets here on GitHub. It makes it easier to have an overview when a GItHub ticket is about only one thing. Don't be afraid to make too many tickets.

Also, I fixed the first one. It will be in the next update.

**Comment by magnusulf** [comment]
Should be fixed in just published 3.1.4

**Comment by magnusulf** [comment]
Also you can now type "faction-France" or "player-France" to specify whether you want to give the permission to the player or faction called "France".
